server/app/services/program/ProgramService.java

```diff
@@ -759,6 +787,22 @@ private void validateLocalizationStatuses(
     }
   }
 
+  private void validateBlockLocalizations(
```

Comment:
**question**: Do we also want to validate that all provided updates map to an existing block? As is I think we just silently drop them on the floor.